packages/material-ui/test/integration/Menu.test.js

```diff
@@ -147,6 +150,28 @@ describe('<Menu> integration', () => {
     });
   });
 
+  describe('render with keepMounted', () => {
+    let wrapper;
+
+    before(() => {
+      // Using render instead of createClientRender because createClientRender specifies an explicit
```

Comment:
Yeah current situation isn't ideal because many tests aren't isolated. Later I would actively block setting `baseElement` since we can't isolate aria scopes anyway and always need to consider the whole document.